services/httpservice/client_test.go

```diff
@@ -152,10 +152,10 @@ func TestIsReservedIP(t *testing.T) {
 		ip net.IP
 		want bool
 	}{
-		{"127.0.0.1", net.IPv4(127, 0, 0, 1), true},
+		{"127.8.3.5", net.IPv4(127, 8, 3, 5), true},
```

Comment:
nit: for sanity, keep `127.0.0.1` as a test case and add the others?

Reply:
The idea was to show case where the difference between the `IsOwnIP` and `IsReservedIP` test, because otherwise we have the same test case actually in both tests, so that might be confusing. 0/5, happy to add it back.